packages/client/src/components/Auth/Signup.jsx

```diff
@@ -46,43 +46,52 @@ export const Signup = () => {
   }
 
   return (
-    <Container maxWidth="xs" sx={{ marginY: '2rem' }}>
-      <Typography variant="h3" textAlign="center" color="white">
-        Create Account
-      </Typography>
-      <Form onSubmit={handleSubmit}>
-        <InputField
-          required
-          type="text"
-          placeholder="enter name"
-          value={fullName}
-          onChange={(e) => setFullName(e.target.value)}
-        />
-        <InputField
-          required
-          type="email"
-          placeholder="enter email"
-          value={email}
-          onChange={(e) => setEmail(e.target.value)}
-        />
-        <InputField
-          required
-          type="password"
-          placeholder="enter password"
-          value={password}
-          onChange={(e) => setPassword(e.target.value)}
-        />
-        <Button type="submit">Create</Button>
-
-        <Typography>
-          Already a member?
-          <Link to="/auth">Login</Link>
+    <Section>
+      <Container maxWidth="xs" sx={{ marginY: '2rem' }}>
+        <Typography variant="h3" textAlign="center" color="white">
+          Create Account
         </Typography>
-      </Form>
-    </Container>
+        <Form onSubmit={handleSubmit}>
+          <InputField
+            required
+            type="text"
+            placeholder="enter name"
+            value={fullName}
+            onChange={(e) => setFullName(e.target.value)}
```

Comment:
why don't you create a **handler** function for all the changed inputs, also, is there a reason you are not using MUI `TextField` instead of `inputField`?

Reply:
Thanks Mohamed!